mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -462,8 +462,7 @@ class _DashboardViewState extends State<DashboardView>
     List<GlobalKey> globalKeys = [];
     final prefs = await SharedPreferences.getInstance();
 
-    if (prefs.getBool(Config.restartTourShowcase) != null &&
-        prefs.getBool(Config.restartTourShowcase) != true) {
+    if (prefs.getBool(Config.restartTourShowcase) != true) {
```

Comment:
@NoahNsimbe Yes, It's supposed to be equal to null. (the null is intentional)
When the app runs for the 1st time, all the preferences are null because they haven't been created yet(that's how we know the app is new), so the showcase runs only when the respective preferences are null. This is the same logic I used for all the other showcase preferences, i.e homePageShowcase, forYouPageShowcase
Check out `line 491` in this file.